.github/workflows/new-release.yml

```diff
@@ -0,0 +1,66 @@
+name: New Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        type: string
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - name: Get version # TODO(eeeqeee) add option to increment from current version
+        id: get-version
+        run: |
+          echo "version=${{ inputs.version }}" >> $GITHUB_OUTPUT
+      - name: Set git config
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+      - name: Change version
+        run: |
+          TEMP_FILE=$(mktemp)
+          jq --arg v "${{ steps.get-version.outputs.version }}" '.version = $v' package.json > $TEMP_FILE
+          cp $TEMP_FILE package.json
+          git add .
+          git commit -m "${{ steps.get-version.outputs.version }}"
```

Comment:
If the version is unset, or the same as prior version, will this still deploy gracefully? Like if we want to redeploy with different secrets, without incrementing version

Reply:
This commit should be empty, but not sure if the release code will fail silently or fail the whole action. Which isn't a huge deal since the deployment would have already happened, but still

Reply:
It probably will fail to commit if its the same since the commit is empty, but I can change it
I can just add a toggle to avoid making a release which would probably be nicer than having part of the action fail